<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainAuthWidget</class>
 <widget class="QWidget" name="MainAuthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainAuthWidget {
            background-image: url(&quot;:/pictures/back.png&quot;);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="WelcomeAuthWidget" name="welcomePage"/>
     <widget class="UserAuthWidget" name="authPage"/>
     <widget class="PsychoResultWidget" name="psychoResults"/>
     <widget class="UserResultLiriWidget" name="userResultLiri"/>
     <widget class="UserResultTorstonWidget" name="userResultTorston"/>
     <widget class="UserTestProtocol" name="userTestProtocol"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WelcomeAuthWidget</class>
   <extends>QWidget</extends>
   <header>common/ui/auth/WelcomeAuthWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserAuthWidget</class>
   <extends>QWidget</extends>
   <header>common/ui/auth/UserAuthWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PsychoResultWidget</class>
   <extends>QWidget</extends>
   <header>common/ui/auth/PsychoResultWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserResultLiriWidget</class>
   <extends>QWidget</extends>
   <header>common/ui/auth/UserResultLiriWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserResultTorstonWidget</class>
   <extends>QWidget</extends>
   <header>common/ui/auth/UserResultTorstonWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserTestProtocol</class>
   <extends>QWidget</extends>
   <header>common/ui/auth/UserTestProtocol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
